README.rst

```diff
@@ -1,131 +1,156 @@
+-  meson >= 0.56.0
+-  ``gcc`` or ``clang``
 
-Optional:
-
--  docutils (for documentation)
-
-Obtaining the source code
-^^^^^^^^^^^^^^^^^^^^^^^^^
-
-Stable releases
-"""""""""""""""
-
-You can find the source tarballs at the
-`Releases <https://github.com/taisei-project/taisei/releases>`__ section on
-Github. **Do not** grab Github's auto-generated source archives, those do not
-contain the required submodules. This only applies for versions v1.3 and above.
+Obtaining Source Code
+^^^^^^^^^^^^^^^^^^^^^
 
-Latest code from git
-""""""""""""""""""""
+You can obtain the source code of the stable releases from the
+`Releases <https://github.com/taisei-project/taisei/releases>`__ page on
+GitHub, alongside the binaries. (Note that you **MUST** use the releases page,
+as the ``Download ZIP`` link on the main repo *does not* include all the code
+necessary to build the game.)
 
-If you cloned Taisei from git, make sure the submodules are initialized:
+You can also grab the latest code from git using the following commands:
 
 ::
 
-    git submodule init
-    git submodule update
+    git clone https://github.com/taisei-project/taisei
+    cd taisei
+    git submodule update --init --recursive
 
-This step needs to be done just once, and can be skipped if you specified the
-``--recursive`` or ``--recurse-submodules`` option when cloning.
+You should also run ``git submodule update`` whenever you pull in
+new code, checkout another branch, or perform any ``git`` actions. The ``pull``
+and ``checkout`` helper scripts can do that for you automatically.
 
-**Important:** You should also run ``git submodule update`` whenever you pull in
-new code, checkout another branch, etc. The ``pull`` and ``checkout`` helper
-scripts can do that for you automatically.
+**Important:** Again, make sure you download the source code from either the
+``Releases`` page *or* using ``git clone``. The ``Download ZIP`` link *will not
+work!*
 
-Compiling from source
+Compiling Source Code
 ^^^^^^^^^^^^^^^^^^^^^
 
-To build and install Taisei on \*nix, just follow these steps:
+Currently, we recommend building Taisei on a \*nix or macOS-based system.
 
-::
-
-    cd /path/to/taisei/source
-    mkdir build
-    cd build
-    meson --prefix=$yourprefix ..
-    ninja
-    ninja install
-
-This will install game data to ``$prefix/share/taisei/`` and build this
-path *statically* into the executable. This might be a package
-maintainer’s choice. Alternatively you may want to add
-``-Dinstall_relative=true`` to get a relative structure like
+While Taisei is highly configurable, the easiest way to compile the code for
+your host machine is:
 
 ::
 
-    $prefix/taisei
-    $prefix/data/
+    meson setup build/
+    meson compile -C build/
+    meson install -C build/
 
-``install_relative`` is always set when building for Windows.
-
-The OpenGL ES 3.0 backend is not built by default. To enable it, do:
+You can also package the game into a ``.zip`` archive.
```

Comment:
I don't think it's necessary to document this here; this is a detail of the release workflow that most users won't care about. If we ever want to document releases, that can go there (along with the other packing targets e.g. tar/txz).